test/suite/test_tiered04.py

```diff
@@ -36,6 +36,8 @@ class test_tiered04(wttest.WiredTigerTestCase):
 
     # If the 'uri' changes all the other names must change with it.
```

Comment:
Maybe generate all of these strings automatically from the uri (or from the base filename)?

Reply:
Changed.